### ISSUE TITLE
Vue.config.productionTip = false not working anymore

### ISSUE THREAD

**Issue by markymark** [vuejs/vue#7562]
### Version
2.5.13

### Reproduction link
[https://jsfiddle.net/chrisvfritz/50wL7mdz/](https://jsfiddle.net/chrisvfritz/50wL7mdz/)

### Steps to reproduce
With all current tools freshly installed, generate a new webpack-based project

### What is expected?
Don't show the (annoying) warning

### What is actually happening?
Warning is shown

As reported by @railstation (and closed automatically) in #7507

**Comment by yyx990803** [comment]
Reproduction please.

**Comment by liquidvisual** [comment]
Can you please reopen this issue. Reproduction here:

https://codepen.io/liquidvisual/pen/eVdKvo?editors=1011

**Comment by shepelevstas** [comment]
I can't find any other way to stop production tip but to directly change it in vue.js file

**Comment by liquidvisual** [comment]
@shepelevstas Same here

**Comment by liquidvisual** [comment]
Codepen demo to show this:

https://codepen.io/liquidvisual/pen/eVdKvo

**Comment by yyx990803** [comment]
fixed in ced00b1d

**Comment by fancydev18** [comment]
@yyx990803  Please support this in vue.config.js too. Now it gives
```
 ERROR  Invalid options in vue.config.js: "productionTip" is not allowed
```
